tensorboard/plugins/debugger_v2/debug_data_multiplexer.py

```diff
@@ -115,6 +115,10 @@ def Runs(self):
                 self._reader.update()
                 # TODO(cais): Start off a reading thread here, instead of being
                 # called only once here.
+            except AttributeError as error:
```

Comment:
Instead of this, what do you think about checking

```py
if tensorflow.__version__ <= "2.1.0":
  return {}
```

Reply:
I'd still like to investigate why the tb-nightly packager failed, and why it was using tensorflow 2.1.0.dev20200102 when it ran on Jan 9.  Then, it would be nice if we gracefully failed when running ToT TB with TF 2.1.0.dev20200102.

So it would make sense to do a check like:
`if parse_version(tf_version) < parse_version("2.1.0.dev20200103"):`

I don't fully understand why, but tf-nightlies are still marked with major/minor "2.1.0", so this check would fail.  That is, "2.1.0" was released before "2.1.0.dev20200103", but parse_version thinks that the former is greater than the latter.

Would it suffice to keep the AttributeError check for now, and have a more descriptive comment that it should be safe to remove when TF 2.2 is released and stable?

Reply:
The change that started around 20200103 does look strange and needs investigation for full understanding. I'll approve the `AttributeError` approach now to unblock builds.

Reply:
> I don't fully understand why, but tf-nightlies are still marked with
> major/minor "2.1.0", so this check would fail. That is, "2.1.0" was
> released before "2.1.0.dev20200103", but parse_version thinks that the
> former is greater than the latter.

TensorFlow 2.1.0 was released on 2020-01-08, after the 2020-01-03
nightly.

The behavior of `parse_version` is correct: `.devWHATEVER` is a
[pre-release specifier, as listed in PEP 440][pep-440].

TensorFlow’s 2.1.0 version bump was just merged into master a few hours
ago, so we should expect tomorrow’s nightly to be `2.2.0.dev20200111`:
<https://github.com/tensorflow/tensorflow/commit/f6404f4f24e756196fd710c27243e10f625ac2e8>

[pep-440]: https://www.python.org/dev/peps/pep-0440/#pre-releases